srv/salt/ceph/openattic/default.sls

```diff
@@ -5,3 +5,13 @@ install openattic:
     - pkgs:
       - openattic
 
+enable openattic-rpcd:
```

Comment:
openattic-rpcd service was already removed from the development branch in openATTIC, so it will not exist in the next openATTIC version.

Also, doesn't the openATTIC RPM installation automatically starts these services?